src/plugins/discover/public/components/doc_table/_doc_table.scss

```diff
@@ -124,3 +124,8 @@
     }
   }
 }
+
+.dscTruncateByHeight {
+  overflow: hidden;
+  display: inline-block;
```

Comment:
Nice!  Yes, looks like `display: inline-block;` fixes the problem.

I think we could move `overflow: hidden` next to `max-height` when it's > 0 and don't add additional styles when it's 0 since `max-height: none;` is a default behaviour.

Reply:
did that. thx